cfme/web_ui/__init__.py

```diff
@@ -3399,7 +3399,8 @@ def get_cell(self, row_text, col_index):
             Selenium element of the cell.
         """
         self.expand_all_sections()
-        cell_loc = ".//div/div[1][contains(., '{}')]/../div[{}]".format(row_text, col_index + 2)
+        cell_loc = ".//div/div[1][contains(., '{}')]//tbody/tr[{}]/td".format(
```

Comment:
@mshriver As you said I tried locally but got the same results then I removed the `root=self.loc` at line number 3404 and ran my test so it moves into next steps means the path matching the dirftGrid but unable to check drift differences and return false.
I got the following error:
`>                   raise SoftAssertionError(_thread_locals.caught_asserts)
E                   SoftAssertionError: 
E                   Drift analysis results are equal when they shouldn't be (cfme/tests/infrastructure/test_host_drift_analysis.py:133)

../../../fixtures/soft_assert.py:62: SoftAssertionError
========================================================== 1 passed, 1 error in 1317.31 seconds ===========================================================`